chevah_build

```diff
@@ -30,7 +30,8 @@ PSUTIL_VERSION="5.8.0"
 SUBPROCESS32_VERSION="3.5.4"
 
 # Versions no longer upgradable because of Python 2 deprecation.
-PIP_VERSION="20.3.4"
+# Backported fix for https://github.com/pypa/pip/issues/9827.
+PIP_VERSION="20.3.4chevah1"
```

Comment:
Where is the code for this release ?

I don't see it at https://github.com/chevah/pip/branches

Reply:
It's at https://github.com/pypa/pip/pull/9827/files, as linked in the PR description. There is no code from me.

Do you want me to re-fork https://github.com/chevah/pip? The current fork is 9 years old.

Reply:
Did a "Fetch upstream" to get our fork up to date. Unfortunately, there's no branch for Python 2.7 compatibility.

What to do? Hard-reset our fork to something that was still compatible and branch out from there? 

At https://pypi.org/project/pip/#history, I see version 20.3.4 was released on January 23, 2021. Corresponding commit would be https://github.com/chevah/pip/commit/a27d06eecf12f16fab7ae78b0e0889745c7a4436.

Reply:
OK, I've got something at https://github.com/chevah/pip/tree/20.3.4chevah. Good enough? Anything else? Thanks.